core/src/main/java/org/mskcc/cbio/portal/dao/DaoReferenceGenome.java

```diff
@@ -103,13 +114,74 @@ public static void deleteAllRecords() throws DaoException {
         }
     }
 
+    /**
+     * Delete Reference Genome by internal ID
+     * @param referenceGenomeId internal ID of reference genome
+     * @throws DaoException Database Error.
+     */
+    public void deleteRecord(int referenceGenomeId) throws DaoException {
```

Comment:
this means all genes are also deleted and hence all studies are deleted...quite drastic measure if all you want is to UPDATE the genome build. Wouldn't it be better to implement an UPDATE statement?

Reply:
good point! Change made.